bin/dotd

```diff
@@ -405,7 +406,10 @@ def main
           title: "DTP (Test > Production: #{test_green_commit})"
         )
         print_database_changes production_pr_number
-        GitHub.open_url GitHub.url(production_pr_number)
+        dtp_auto_merged = should_i "merge DTP PR#{production_pr_number}" do
+          GitHub.merge_pull_request(production_pr_number, "DTP (Test > Production: #{test_green_commit})")
+        end
+        GitHub.open_url GitHub.url(production_pr_number) unless dtp_auto_merged
```

Comment:
or if it's easier maybe the non-development check should just be moved inside `open_url`